joomla/joomla-cms
# [Improvement] - Publishing up/down for Menu Items

Pull Request for Issue #17678 .

### Summary of Changes
add publish_up, publish_down for (site) menu items


### Testing Instructions
0. apply pr
1. fix database
2.
set publish_up, publish_down and check that (site) menu item are showed o not
accordingly to the publish_up, publish_down settings

![screenshot from 2017-08-25 08-20-07](https://user-images.githubusercontent.com/181681/29701792-416f3688-896e-11e7-841c-d48930135c9c.png)

### Expected result
you are able to set  publish_up, publish_down for (site) menu items


### Actual result
no feature


### Documentation Changes Required
?
